.github/header-checker-lint.yml

```diff
@@ -0,0 +1,15 @@
+{"allowedCopyrightHolders": ["Google LLC"],
```

Comment:
Needs to be "pandas-gbq Authors", since this was not originally Google-authored.